docs/reference/contrib/modeladmin/migrating_to_snippets.md

```diff
@@ -0,0 +1,141 @@
+
+    - `ModelAdmin.thumb_image_field_name`
+    - `ModelAdmin.thumb_image_width`
+    - `ModelAdmin.thumb_classname`
+    - `ModelAdmin.thumb_col_header_text`
+    - `ModelAdmin.thumb_default`
+
+### Custom CSS and JS
+
+ModelAdmin supports inserting custom extra CSS and JS files into the admin via the following attributes on the ModelAdmin class:
+
+- `ModelAdmin.index_view_extra_css`
+- `ModelAdmin.index_view_extra_js`
+- `ModelAdmin.form_view_extra_css`
+- `ModelAdmin.form_view_extra_js`
+- `ModelAdmin.inspect_view_extra_css`
+- `ModelAdmin.inspect_view_extra_js`
+
+This is not supported in snippets, but custom CSS and JS files can be inserted by overriding the respective view's template.
+
+### Helper classes
+
+Helper classes encapsulate the logic that is commonly used across views in ModelAdmin. These classes do not exist for snippets, as the similar logic now relies on generic patterns used across Wagtail.
+
+- `ModelAdmin.url_helper_class`
+
+  The base {class}`~wagtail.admin.viewsets.base.ViewSet` class has {meth}`~wagtail.admin.viewsets.base.ViewSet.get_urlpatterns()` and {meth}`~wagtail.admin.viewsets.base.ViewSet.get_url_name()` methods that can be used to manage the URLs of snippets views. The URL names can be used with Django's `reverse()` function to generate URLs.
+
+- `ModelAdmin.permission_helper_class`
+
+  Wagtail uses an internal permission policy system to manage permissions across the admin. The {class}`~SnippetViewSet` class has a {attr}`~SnippetViewSet.permission_policy` attribute, which is an instance of a permission policy class.
+
+- `ModelAdmin.button_helper_class`
+
+  The pre-existing [`register_snippet_listing_buttons`](register_snippet_listing_buttons) and [`construct_snippet_listing_buttons`](construct_snippet_listing_buttons) hooks can be used to customise the buttons in the listing view. For other views, custom buttons can be added by overriding the respective view's template.
+
+- `ModelAdmin.search_handler_class`
+  When searching snippets, Wagtail's default search backend is used. To use a different backend, the {attr}`~SnippetViewSet.search_backend_name` attribute can be overridden. If the attribute is set to `None`, the search uses the Django ORM instead.
+
+  As the `search_handler_class` attribute is not supported in snippets, the `ModelAdmin.extra_search_kwargs` attribute is also not supported.
+
+### Other customisations
+
+- `ModelAdmin.empty_value_display` and `ModelAdmin.get_empty_value_display()`
+
+  This can be replaced by the Django-standard {meth}`~django.db.models.Model.get_FOO_display` method on the model.
+
+- `ModelAdmin.get_ordering(request)`
+
+  The {attr}`SnippetViewSet.ordering` attribute is responsible for the default ordering of the index view. The index view handles per-request ordering based on the columns that are specified in `list_display`. For more advanced customisation, you can override the {attr}`~SnippetViewSet.index_view_class`.
```

Comment:
Is it worth calling out Django's meta ordering also? I assume that this would be the simplest way to order queries?

Reply:
I suppose so. Updated.